lib/signaling/v2/publisherhintsignaling.js

```diff
@@ -0,0 +1,103 @@
+/* eslint callback-return:0 */
+'use strict';
+
+const MediaSignaling = require('./mediasignaling');
+
+class PublisherHintsSignaling extends MediaSignaling {
+  /**
+   * Construct a {@link RenderHintsSignaling}.
+   */
+  constructor(getReceiver, options) {
+    super(getReceiver, 'publisher_hints', options);
+    this.on('ready', transport => {
+      this._log.info('publisher_hints transport ready: ', transport);
+      transport.on('message', message => {
+        this._log.debug('publisher_hints Incoming: ', message);
```

Comment:
```suggestion
```

Reply:
@manjeshbhargav - I have added this back, since its also been found very useful. We also log incoming and outgoing messages for subscriber hints.